backend/test/test_github_app_service.py

```
@@ -0,0 +1,271 @@
+	mock_client = MagicMock()
+	mock_client.get = AsyncMock(return_value=mock_response)
+
+	with patch("services.github_app_service.generate_jwt", return_value="jwt"), \
+		patch("services.github_app_service.get_installation_access_token",
+		new_callable=AsyncMock,
+		return_value={"token": "fake_token"}), \
+			patch("services.github_app_service.httpx.AsyncClient") as mock_class:
+		
+		mock_class.return_value.__aenter__ = AsyncMock(return_value=mock_client)
+		mock_class.return_value.__aexit__ = AsyncMock(return_value=None)
+		with pytest.raises(Exception):
```

Ruff (0.15.11)

[warning] Do not assert blind exception: `Exception`

(B017)